src/components/editor/editor.tsx

```diff
@@ -118,6 +116,8 @@ export class Editor implements ComponentInterface {
       return;
     }
 
+    injectHeadCSS(this.containerRef.getRootNode());
```

Comment:
unfortunately the css should not be a children of the editable container

only html elements of content can be children of the contenteditable

sorry it's tricky

Reply:
its not a children of the edtiable container, it's a children of the shadowroot where the edtiable container is in

Reply:
I missed the `.getRootNode`, alright